grpcclient/grpcclient.go

```diff
@@ -128,5 +137,13 @@ func (cfg *Config) DialOption(unaryClientInterceptors []grpc.UnaryClientIntercep
 			Timeout:             time.Second * 10,
 			PermitWithoutStream: true,
 		}),
+		grpc.WithInitialConnWindowSize(int32(cfg.InitialConnectionWindowSize)),
+		grpc.WithInitialWindowSize(int32(cfg.InitialStreamWindowSize)),
```

Comment:
See here:
https://github.com/grafana/mimir/issues/4010#issuecomment-1396936200

I think that once you set it, you automatically disable the BDP estimator, which is something we don't want.